### ISSUE TITLE
Is this still necessary?

### ISSUE THREAD

**Issue by MichaelMMeskhi** [openml/openml-tutorial#1]
Is this something that would benefit lots of new OpenML users? If so, I can start documenting my experience with OpenML and convert them into tutorials.

**Comment by joaquinvanschoren** [comment]
Yes, please. This tutorial is how we introduce people to OpenML in workshops. You are very welcome to add to it. 

You can also add examples in the OpenML-Python API documentation:
https://openml.github.io/openml-python/master/examples/index.html
